blaze/server/tests/test_server.py

```diff
@@ -385,6 +385,63 @@ def test_server_accepts_non_nonzero_ables():
     Server(DataFrame())
 
 
+def serialize_query_with_map_builtin_function(test, serial):
+    """
+    serialize a query that invokes the 'map' operation using a builtin function
+    return the result of the post operation along with expected result
+    """
+    t = symbol('t', discover(iris))
+    expr = t.species.map(len, 'int')
+    query = {'expr': to_tree(expr)}
+    response = test.post('/compute',
+                         data=serial.dumps(query),
+                         headers=mimetype(serial))
+    assert 'OK' in response.status
+    respdata = serial.loads(response.data)
+    result = serial.data_loads(respdata['data'])
+
+    exp_res = compute(expr, {t: iris}, return_type=list)
+    return (exp_res, result)
+
+
+@pytest.mark.parametrize('serial', most_formats)
+def test_map_client_server(iris_server, serial):
+    """
+    serialization for 'most_formats' returns a list so this is valid for
+    everything in most_formats
+    """
+    exp_res, result = serialize_query_with_map_builtin_function(iris_server,
+                                                                serial)
+
+    assert result == exp_res
+
+
+def test_map_client_server_fastmsgpack(iris_server):
+    """
+    serialization using fastmsgpack returns a Series object so our assertion
+    must be for all elements in Series
+    """
+    exp_res, result = serialize_query_with_map_builtin_function(iris_server,
+                                                                fastmsgpack)
+
+    assert all(result == exp_res)
+
```

Comment:
> could we catch only the error we are expecting?

break apart 'fastmsgpack' test from `test_map_client_server()` into `test_map_client_server_fastmsgpack()`. The result of fastmsgpack is a Series object as opposed to a list so this makes the assertions cleaner.